src/modules/person/persistence/person.repository.ts

```diff
@@ -74,10 +88,31 @@ export function mapEntityToAggregateInplace(entity: PersonEntity, person: Person
 
 @Injectable()
 export class PersonRepository {
+    public readonly ROOT_ORGANISATION_ID: string;
+
     public constructor(
         private readonly kcUserService: KeycloakUserService,
         private readonly em: EntityManager,
-    ) {}
+        config: ConfigService<ServerConfig>,
+        private readonly dBiamPersonenkontextRepo: DBiamPersonenkontextRepo,
+    ) {
+        this.ROOT_ORGANISATION_ID = config.getOrThrow<DataConfig>('DATA').ROOT_ORGANISATION_ID;
+    }
+
+    public async getPersonScopeWithPermissions(permissions: PersonPermissions): Promise<PersonScope> {
```

Comment:
Please change to `private`. The scope is tightly coupled to mikro-orm and should not be accessible outside of the repo.

Reply:
Done.